src/codemagic/tools/_app_store_connect/action_groups/beta_groups_action_group.py

```diff
@@ -0,0 +1,111 @@
+from typing import Sequence
+from typing import Set
+from typing import Tuple
+from typing import Union
+
+from codemagic import cli
+from codemagic.apple import AppStoreConnectApiError
+from codemagic.apple.resources import BetaGroup
+from codemagic.apple.resources import Build
+from codemagic.apple.resources import LinkedResourceData
+from codemagic.apple.resources import ResourceId
+from codemagic.cli import Colors
+
+from ..abstract_base_action import AbstractBaseAction
+from ..action_group import AppStoreConnectActionGroup
+from ..arguments import BuildArgument
+from ..errors import AppStoreConnectError
+
+
+class BetaGroupsActionGroup(AbstractBaseAction):
+
+    @cli.action('add-build',
+                BuildArgument.BUILD_ID_RESOURCE_ID_REQUIRED,
+                BuildArgument.BETA_GROUP_NAMES_REQUIRED,
+                action_group=AppStoreConnectActionGroup.BETA_GROUPS)
+    def add_build_to_beta_groups(self, build_id: Union[ResourceId, Build], beta_group_names: Sequence[str]):
```

Comment:
Is there any benefit in allowing `Build` instances as argument, like fewer API calls? It would simplify the logic here if only identifiers were allowed.

Applies to similar thing in `remove_build_from_beta_groups` too.

Reply:
Signature is not updated in `AbstractBaseAction`.